setup.py

```diff
@@ -19,6 +19,7 @@
     author="Pavel Dedík",
     author_email="pavel.dedik@kiwi.com",
     packages=find_packages(),
+    python_requires=">=3",
```

Comment:
3.5 if that matters/possible. What breaks Python 2 are type annotations, which were added in 3.5